tests/test_utils.py

```diff
@@ -423,3 +428,15 @@ def test_drop_extras(from_line, given, expected):
         assert ireq.markers is None
     else:
         assert str(ireq.markers).replace("'", '"') == expected.replace("'", '"')
+
+
+def test_get_pip_version_for_python_executable():
+    result = get_pip_version_for_python_executable(sys.executable)
+    assert Version(pip.__version__) == result
+
+
+def test_get_sys_path_for_python_executable():
+    result = get_sys_path_for_python_executable(sys.executable)
+    # not testing for equality, because pytest adds extra paths into current sys.path
+    for path in result:
```

Comment:
`assert result` should be checked to make sure that below assert would be reached.

Reply:
good point, added an assert